src/definitions_cxx.hpp

```diff
@@ -27,6 +27,8 @@
 
 #define AUTOMATED_TESTER_ENABLED (0 && ALPHA_OR_BETA_VERSION)
 
+#define MATRIX_DRIVER_DEBUG 0
+
```

Comment:
This could be plumbed in to the build system - look for how `ENABLE_SYSEX_LOAD` works for an example.

Reply:
Removed the gremlin. I can see about the build system element.